fastlane/storekit_config_file_watcher.rb

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+# Get arguments
+bundle_id = ARGV[0]
+storekit_config_source = ARGV[1]
+
+unless bundle_id && storekit_config_source
+  puts "Usage: ruby storekit_config_file_watcher.rb <bundle_id> <storekit_config_path>"
+  exit 1
+end
+
+unless File.exist?(storekit_config_source)
+  puts "Error: StoreKit config not found at #{storekit_config_source}"
+  exit 1
+end
+
+def get_storekit_paths(bundle_id)
+  # Get the app container path
+  container_path = `xcrun simctl get_app_container booted "#{bundle_id}"`.strip
+  return nil unless $?.success?
+
+  # Extract base container path
+  match = container_path.match(/(.*\/data\/Containers)/)
+  return nil unless match
+  base_container_path = match[1]
+
+  # Find the StoreKit app group container
+  app_group_pattern = "#{base_container_path}/Shared/AppGroup/*"
+  app_group_dirs = Dir.glob(app_group_pattern)
+
+  storekit_app_group = app_group_dirs.find do |app_group_dir|
+    metadata_plist = File.join(app_group_dir, '.com.apple.mobile_container_manager.metadata.plist')
+    next false unless File.exist?(metadata_plist)
+
+    begin
+      identifier = `plutil -extract MCMMetadataIdentifier raw -o - "#{metadata_plist}"`.strip
+      identifier == 'group.com.apple.storekit'
+    rescue
+      false
+    end
+  end
+
+  return nil unless storekit_app_group
+
+  octane_path = File.join(storekit_app_group, 'Documents', 'Persistence', 'Octane')
+  storekit_dir = File.join(octane_path, bundle_id, 'Configuration.storekit')
+  storekit_file = File.join(storekit_dir, 'Configuration.storekit')
```

Comment:
All this logic already exists in the `setup_simulator_storekit_config` action. So I'd try to avoid this duplication and always use that action from here. Is there any limitation to do that?